os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -271,6 +271,16 @@ static void up_dumpstate(void)
 	/* Show interrupt stack info */
 
 	lldbg("sp:     %08x\n", sp);
+	if (current_regs) {
+		lldbg("*******************************************\n");
+		lldbg("Asserted during Interrupt mode\n");
```

Comment:
This info is important. Could you make TRAP showing this well?

Reply:
I have updated TRAP tool as per the latest assert logs.